src/modules/feature-modules/assessment/pages/innovations/innovations-list.component.ts

```diff
@@ -48,7 +48,7 @@ export class InnovationsListComponent extends CoreComponent implements OnInit {
 
   form = new FormGroup(
     {
-      search: new FormControl('', { updateOn: 'change' }),
+      search: new FormControl(''),
```

Comment:
Should this be `updateOn: 'blur'` like the others?

Reply:
Since we defined the form as `updatedOn: blur`, the defaults for his "childs" will be the same unless we say otherwise (e.g., `assignedToMe`).